pkg/apis/internal.admin.acorn.io/v1/resources.go

```diff
@@ -84,14 +91,14 @@ func (current *Resources) Fits(incoming Resources) error {
 
 	// Define function for checking int resources to keep code DRY
 	checkResource := func(resource string, currentVal, incomingVal int) {
-		if currentVal < incomingVal {
+		if currentVal != Unlimited && currentVal < incomingVal {
 			exceededResources = append(exceededResources, resource)
 		}
 	}
 
 	// Define function for checking quantity resources to keep code DRY
 	checkQuantityResource := func(resource string, currentVal, incomingVal resource.Quantity) {
-		if currentVal.Cmp(incomingVal) < 0 {
+		if !currentVal.Equal(UnlimitedQuantity()) && currentVal.Cmp(incomingVal) < 0 {
```

Comment:
Personally, I would like to find a way to get rid of `UnlimiteQuantity`. Can `CmpInt64(Unlimited)` be used here instead?

Reply:
Sure, I can do that.

Do you want to get rid of the `UnlimitedQuantity` function entirely? I mainly made it to make setting unlimited quantities more straight forward. Without it there would be a lot of `resource.NewQuantity(Unlimited,  resource.DecimalSI)` set. Is that something we'd want as it is more explicit?

Reply:
No, I think it's fine then. Thanks for the explanation.